database/localhost/docker-compose.yaml

```diff
@@ -17,3 +17,17 @@ services:
     restart: always
     ports:
       - 8081:8080
+
+  mongo:
+    image: mongo:latest
+    container_name: tilde-mongodb
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: password
+      MONGO_INITDB_DATABASE: root-db
+    volumes:
+      - ./gitignore/mongodb:/var/lib/mongodb/data
+      - ./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
```

Comment:
I'm not convinced that these volumes are as they should be.
Firstly:
```
      - ./gitignore/mongodb:/var/lib/mongodb/data
```
Are you sure that mongo stores it's data in `/var/lib/mongodb/data` ? I dont think it does.  Take a look at the mondo image documerntation to see what volumnes it suggests: https://hub.docker.com/_/mongo  

Also, what is this line for?
```
      - ./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
```
If you look inside the directory it just has sql stufff. https://github.com/Umuzi-org/Tilde/tree/develop/database/localhost/docker-entrypoint-initdb.d 

And I expect mongo will completely ignore it anyway, unless it's mimicking the postgres image for some reason. 

It's really not good to make guesses with this kind of thing.